servers/cromwell/jobs/controllers/jobs_controller.py

```diff
@@ -47,10 +51,14 @@ def update_job_labels(id, body):
 
     :rtype: UpdateJobLabelsResponse
     """
+    auth_token = request.headers.get('Authentication')
```

Comment:
It might be nice to break these two lines into a function that returns auth_token and auth, or even a decorator so you don't have to repeat as much. This might also make it possible to write tests for the auth code.